.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,37 @@
+name: Changelog
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  generate_changelog:
+    runs-on: ubuntu-latest
+    name: Generate changelog for master branch
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Generate changelog
+        uses: charmixer/auto-changelog-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Commit files
+        env:
+          CI_USER: iMicknl
+          CI_EMAIL: mick@imick.nl
```

Comment:
Need a secret here.

Reply:
this info is public though, took it from the public github profile.